docs/pbc/all/request-for-quote/use-cases-request-for-quote-processes.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Use cases: Request for Quote processes"
+description: This document describes the use cases for Requests for Quote.
+last_updated: Nov 15, 2022
+template: concept-topic-template
+---
+
+
+## Use case 1: Long waiting time until Request for Quote (RFQ) gets processed
+
+The [request for quote](/docs/pbc/all/request-for-quote/request-for-quote.html) process is currently not entirely efficient because it is manual. As a result, sometimes, customers have to wait for a long time until their RFQ gets approved or denied. 
```

Comment:
```suggestion
The [Request for Quote](/docs/pbc/all/request-for-quote/request-for-quote.html) process is currently not fully efficient because it is manual. As a result, sometimes customers have to wait a long time until their Request for Quote (RFQ) gets approved or denied. 
```
Same as above, if its a specific term then capitalised letters if not then no.

Reply:
I've adjusted your comment a bit.